packages/e2e/cypress/e2e/app/pivotTables.cy.ts

```diff
@@ -0,0 +1,73 @@
+        );
+
+        // Create a pivot table
+        cy.contains('placed');
+        cy.contains('shipped');
+
+        cy.contains('False');
+        cy.contains('2018-04-02');
+        cy.contains('$236.21');
+    });
+    it.skip('Can create a pivot table chart on explore', () => {
+        // Navigate to the explore page
+        cy.visit(
+            `/projects/${SEED_PROJECT.project_uuid}/tables/orders?create_saved_chart_version=%7B%22tableName%22%3A%22orders%22%2C%22metricQuery%22%3A%7B%22exploreName%22%3A%22%22%2C%22dimensions%22%3A%5B%22orders_order_date_week%22%2C%22orders_status%22%2C%22orders_is_completed%22%5D%2C%22metrics%22%3A%5B%22orders_total_order_amount%22%5D%2C%22filters%22%3A%7B%7D%2C%22sorts%22%3A%5B%7B%22fieldId%22%3A%22orders_order_date_week%22%2C%22descending%22%3Atrue%7D%5D%2C%22limit%22%3A500%2C%22tableCalculations%22%3A%5B%5D%2C%22additionalMetrics%22%3A%5B%5D%2C%22metricOverrides%22%3A%7B%7D%7D%2C%22tableConfig%22%3A%7B%22columnOrder%22%3A%5B%22orders_order_date_week%22%2C%22orders_status%22%2C%22orders_is_completed%22%2C%22orders_total_order_amount%22%5D%7D%2C%22chartConfig%22%3A%7B%22type%22%3A%22table%22%2C%22config%22%3A%7B%22showColumnCalculation%22%3Afalse%2C%22showRowCalculation%22%3Afalse%2C%22showTableNames%22%3Atrue%2C%22showResultsTotal%22%3Afalse%2C%22showSubtotals%22%3Afalse%2C%22columns%22%3A%7B%7D%2C%22hideRowNumbers%22%3Afalse%2C%22conditionalFormattings%22%3A%5B%5D%2C%22metricsAsRows%22%3Afalse%7D%7D%7D`,
+        );
+
+        cy.contains('Configure').click();
+        cy.contains('Drag dimensions into this area to pivot your table');
+
+        const dragSelector =
+            '[role="tabpanel"] [data-rfd-draggable-id="orders_is_completed"]';
+        const dropSelector = '[data-rfd-droppable-id="COLUMNS"]';
+
+        // Drag and drop is not working as expected,
+        cy.drag(dragSelector, dropSelector);
```

Comment:
I really wanted to make myself a pivot chart from a chart in explore, but for that we need to drag/drop the column, I couldn't get this drag and drop on cypress to work, but I'm leaving here the code so I can try some other day.